### ISSUE TITLE
Alpha is situational at best

### ISSUE THREAD

**Issue by stelscarrot** [CleverRaven/Cataclysm-DDA#53110]
### Is your feature request related to a problem? Please describe.

Alpha threshold is worse than useless. Malicious, even. As if made to troll new players. When it is described as mutation that makes character "perfect candidate to tidy this mess", it makes character weaker or same in most cases, and stronger in the best case scenario, and even then it is bad.
The stats go:
| STR,DEX,INT,PER | No threshold max | Alpha |
| ------------- | ------------- | ------------- |
| 4 4 4 4 | 8 8 8 8 | 12 11 11 12  |
| 6 6 6 6 | 10 10 10 10 | 13 12 12 12  |
| 8 8 8 8 | 12 12 12 12 | 14 13 13 14  |
| 10 10 10 10 | 14 14 14 14 | 15 14 14 15  |
| 12 12 12 12 | 16 16 16 16 | 16 15 15 16  |
| 14 14 14 14 | 18 18 18 18 | 17 16 16 17  |

Only from 4 to 8 stat boost bigger than 1 for STR and PER, with DEX and INT being only 4 to 6. And after some point it even worse than without threshold. It could be beneficial for 4 4 4 4 character to go Alpha if they have ~10 alpha serums on their hands. In other case there are others thresholds, that while not give as much of stats overall, make one stat equal to Alphas, or even bigger, with addition of a lot of unique mutations. 
And this is for 4 4 4 4 character, with one mutagen crafted from 12 another and purifier.
Alpha threshold creates an aura of strength, importance and power. But for average not handicapped character it gives nothing, with additional bonus - you can't take another threshold. 

### Solution you would like.

Make Prime *stat* not stage of stat additions, but a mutation that requires said stat. Like it made with Bird's "Avian Eyes".

### Describe alternatives you have considered.

Additional mutations to alpha threshold, or rework of current. Less sleep is great, but it is not even unique, so Elf-a is inherently better right now.
For instance - make it so mutations would not only give stats, but also protect part of it. So it would be impossible for stat to go bellow amount stat mutation gives, or part of final amount of stat, or fixed amount. It would be cool ability, that could prevent 0 0 0 0 situation, achieved because of poor planning, or unforeseen event.

**Comment by Venera3** [comment]
There's only so much you can bring out of the human form. If Alpha's flavor is supposed to be "human, but better", I don't see them as the strongest, fastest, or err...perceptionistiest of all lines to be honest. Some mild stat boosts, maybe, but much more about emphasizing tribe building and learning - social buffs, a trait to increase theoretic knowledge's skill learning bonus, maybe a chance to learn non-autolearned recipes while crafting with the relevant skill, stuff like that.

**Comment by stelscarrot** [comment]
Yep, instead of taking starting stats of character it taking current. So with webbed hands, muscle consolidation and some purifier you could turn 4 strength into 6, then into 8 and so on. I think I would make separate issue for that.

**Comment by stelscarrot** [comment]
> There's only so much you can bring out of the human form. If Alpha's flavor is supposed to be "human, but better", I don't see them as the strongest, fastest, or err...perceptionistiest of all lines to be honest. Some mild stat boosts, maybe, but much more about emphasizing tribe building and learning - social buffs, a trait to increase theoretic knowledge's skill learning bonus, maybe a chance to learn non-autolearned recipes while crafting with the relevant skill, stuff like that.

Sounds great, but lacks a bit of oomph to be wanted threshold just from that.

**Comment by stelscarrot** [comment]
> There's only so much you can bring out of the human form.

Kinda loosely related to topic, but in a lot of things humans are the perfect form. Most things packed under intelligence in-game are considered human traits. Perception is combination of hearing and sight, which while can't be described as humans strongest points, also not the weakest, losing only to avian sight or bat hearing, and Birds in-game receive both "Extremely Perceptive" and "Avian Eyes", giving them 11 perception, that lower than alpha's 8. There are senses that humans could losing with, like smell and taste, but they are doesn't count towards PER. Dexterity could be a strong case, because people generally slower runners, but then again there is such thing as fine motor skills, that counts as DEX in-game. 
Strength is only stat that can't be argued there.

**Comment by stelscarrot** [comment]
But this is the thing - it doesn't give any boost to non-(severely)handicapped character, as there are non-threshold mutations that give more. And unless "Extremely" traits "visibility" score would count towards exodii hate it still would be better not to take alpha.
I can't even fathom why would it be popular, when it have 1 functioning trait that is less sleep, and because of #53111 it is at the moment gives no boost even to handicapped characters. 
And even if it shouldn't be best, it could be still worst in every field with 200% or in some cases even 300% boost.

**Comment by stelscarrot** [comment]
Main point being - if it is supposed to be stat only threshold it should give at least some stats. And if it shouldn't be stat only threshold it should have something beside them.

**Comment by stelscarrot** [comment]
> If non-threshold mutations are getting your stats to 18 across the board, I hazard that the problem is that there are too many non-threshold stat boosting mutations. Those are high stats for a non-threshold mutant.

Oh, no, of course not. It is only stats that conflict with "Prime" traits. With others you could get to mid 20 in some cases.

**Comment by stelscarrot** [comment]
On the sidenote - moving some stages of stat upgrades to threshold would work too. Actually, it would make some other thresholds more competitive, and diversify them good bit.
